clusterloader2/cmd/clusterloader.go

```diff
@@ -255,6 +259,13 @@ func main() {
 		klog.Exitf("Flag parse failed: %v", err)
 	}
 
+	if profiling {
```

Comment:
If `profiling` is `false` does that mean that pprof won't run?

Reply:
Yes, the pprof runs logic only when /debug/pprof is accessed